### PR TITLE
update nodepool config, add f27, update rhel7.3 to rhel7.4

### PR DESCRIPTION
**Pull request by werwty**

### CODE REVIEW

File: ci/nodepool/nodepool.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-script-dir: /home/pulp/pulp-ci/ci/nodepool/scripts/
+script-dir: /home/pulp/pulp_packaging/ci/nodepool/scripts/
```
Comment (Ichimonji10) [inline ci/nodepool/nodepool.yaml]: Is this a regression?

File: ci/nodepool/nodepool.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 # empty dir, stops nodepool from throwing a traceback on start
 images-dir: /etc/nodepool/images
 
 dburi: 'postgresql:///nodepool'
 
 zmq-publishers:
-  - tcp://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com:8881
+  - tcp://pulp-jenkins.rhev-ci-vms.eng.rdu2.redhat.com:8888
```
Comment (Ichimonji10) [inline ci/nodepool/nodepool.yaml]: Has Jenkins' ZeroMQ configuration changed?

File: ci/nodepool/nodepool.yaml (hunk 1 above)
Comment (werwty) [reply ci/nodepool/nodepool.yaml]: kinda? I would guess that the mass rename accidentally got this one. But we have not been using this file to keep our nodepool up to date so nothing's broken.

File: ci/nodepool/nodepool.yaml (hunk 2 above)
Comment (werwty) [reply ci/nodepool/nodepool.yaml]: I am unsure of when the Jenkins configuration change actually occured. I grabbed this from the current config version on nodepool. 
I'll push an update separating out the commit to update this config to what's currently in nodepool, and the f27 changes I'm proposing.

File: ci/nodepool/nodepool.yaml (hunk 1 above)
Comment (Ichimonji10) [reply ci/nodepool/nodepool.yaml]: Hunh. I thought we weren't using a repository named  `pulp_packaging` at all.

Comment (werwty) [reply ci/nodepool/nodepool.yaml]: The repository was renamed from `pulp_packaging` to `pulp_ci`.
The directory on the nodepool server was created when the repo was name `pulp_packaging`.
I am not sure if there's any other things on the nodepool server that are hardcoded to the `/home/pulp/pulp_packaging` location so I have not renamed the directory. But I have it's git remote url to point to `pulp-ci`

Comment (Ichimonji10) [reply ci/nodepool/nodepool.yaml]: Ahh, I understand now. Thanks.

Comment (werwty) [reply ci/nodepool/nodepool.yaml]: I asked @pcreech about this and he is 95% sure nothing else is using the pulp_packaging path, so I will rename it on the server.

**Comment by pcreech** [comment]
Thanks!

**Pull request merged**